syntax = "proto3";

import "data_model_config.proto";

message BodyConfig {
    map<string, DataObjectConfig> body = 1;
}
